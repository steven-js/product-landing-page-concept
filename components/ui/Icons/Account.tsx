const Account = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="16"
      height="22"
      fill="none"
      viewBox="0 0 16 22"
    >
      <path
        fill="var(--neutral-neutral-1000)"
        d="M8 2a3.75 3.75 0 1 1 0 7.5A3.75 3.75 0 0 1 8 2ZM8 .5A5.25 5.25 0 1 0 8 11 5.25 5.25 0 0 0 8 .5Zm7.5 21H14v-3.75A3.75 3.75 0 0 0 10.25 14h-4.5A3.75 3.75 0 0 0 2 17.75v3.75H.5v-3.75a5.25 5.25 0 0 1 5.25-5.25h4.5a5.25 5.25 0 0 1 5.25 5.25v3.75Z"
      />
    </svg>
  )
}

export default Account
