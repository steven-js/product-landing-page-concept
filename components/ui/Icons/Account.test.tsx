import { render } from '@testing-library/react'
import Account from './Account'

describe('Account icon test', () => {
  it('should render the Account icon unchanged', () => {
    const { container } = render(<Account />)
    expect(container).toMatchSnapshot()
  })
})
