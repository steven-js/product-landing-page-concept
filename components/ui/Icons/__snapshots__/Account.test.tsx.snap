// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account icon test should render the Account icon unchanged 1`] = `
<div>
  <svg
    fill="none"
    height="22"
    viewBox="0 0 16 22"
    width="16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M8 2a3.75 3.75 0 1 1 0 7.5A3.75 3.75 0 0 1 8 2ZM8 .5A5.25 5.25 0 1 0 8 11 5.25 5.25 0 0 0 8 .5Zm7.5 21H14v-3.75A3.75 3.75 0 0 0 10.25 14h-4.5A3.75 3.75 0 0 0 2 17.75v3.75H.5v-3.75a5.25 5.25 0 0 1 5.25-5.25h4.5a5.25 5.25 0 0 1 5.25 5.25v3.75Z"
      fill="var(--neutral-neutral-1000)"
    />
  </svg>
</div>
`;
